tsconfig.base.json

```diff
@@ -1,24 +1,35 @@
 {
 	"compilerOptions": {
-		"strict": true,
-		"moduleResolution": "node",
-		"module": "commonjs",
-		"target": "es2020",
-		"lib": ["esnext", "esnext.array", "esnext.asynciterable", "esnext.intl", "esnext.symbol"],
+		"allowSyntheticDefaultImports": true,
+		"alwaysStrict": true,
 		"declaration": true,
 		"declarationMap": true,
-		"sourceMap": true,
-		"removeComments": false,
-		"alwaysStrict": true,
-		"pretty": true,
+		"emitDecoratorMetadata": true,
+		"esModuleInterop": true,
+		"experimentalDecorators": true,
+		"forceConsistentCasingInFileNames": true,
```

Comment:
We don't use this anywhere else.

Reply:
Any reason why we shouldn't use it now? It doesn't hurt 👀 https://www.typescriptlang.org/tsconfig#forceConsistentCasingInFileNames

Reply:
I would assume you'll never really run into that with auto-import, but I guess you have a point.